packages/core/src/components/navbar/navbar.md

```diff
@@ -40,3 +40,35 @@ Wrap your `.pt-navbar-group`s in an element with your desired `width` and `margi
 horizontally center it.
 
 @css pt-navbar.pt-container
+
+@## JavaScript API
+
+The `Navbar` component is available in the __@blueprintjs/core__ package.
+Make sure to review the [general usage docs for JS components](#blueprint.usage).
+
+These components are simple wrappers around the corresponding CSS APIs. Each of them supports the full range of HTML props.
+
+```tsx
+<Navbar>
+    <div style="margin: 0 auto; width: 480px;"> <!-- ADD ME -->
```

Comment:
wait why is this is necessary? undocumented awkwardness. 
maybe `<Navbar innerWidth={480}>`?

Reply:
depends whether you want a flexible width navbar or a fixed width one with full bleed background. I'm inclined to leave this off the API for now, and maybe remove that from the current example

Reply:
Removed